mbed-os.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#b6e5a0a8afa34dec9dae8963778aebce0c82a54b
+https://github.com/ARMmbed/mbed-os/#5941d1718339116cd12914238ec331c84da3d08f
```

Comment:
Add newline at the EOF

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/2f96047ab8eb914c0ce2ee6805a598a1df3445e4).